internal/plugin/server_test.go

```diff
@@ -131,7 +131,8 @@ func TestCDIAllocateResponse(t *testing.T) {
 		},
 	}
 
-	for _, tc := range testCases {
+	for i := range testCases {
```

Comment:
@elezar I can limit the changeset to this fix given we don't really need the `math/rand` lint fix